armedilla_alliyah_samantha/index.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Armedilla, Alliyah Samantha</title>
-    <link href="./mystyle.css" rel="stylesheet" />
+    <link href="./style.css" rel="stylesheet" />
   </head>
   <body>
     <header class="header">
```

Comment:
Also you can add 1 extra new line for readability purpose.

https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g25d18931cb7_3_0